### PR TITLE
feeshy's fucking cursed donator plushie.

### PR DESCRIPTION
**Pull request by capsaicinz**
## About The Pull Request

fuck this. fuck this plushie, fuck everything about it, fuck everything to do with it, fuck anything that WILL do with it.

## How This Contributes To The Skyrat Roleplay Experience

it took years off my life.

## Changelog
:cl:
add: added odd lizzy plushie.
/:cl:

del: removed my will to live to age 30.

### CODE REVIEW

**Comment by Kapu1178** [comment]
post the sprite post the sprite post the sprite post the sprite post the sprite post the sprite post the sprite post the sprite post the sprite post the sprite post the sprite post the sprite post the sprite post the sprite post the sprite post the sprite post the sprite post the sprite post the sprite post the sprite post the sprite post the sprite post the sprite post the sprite post the sprite post the sprite post the sprite post the sprite

**Comment by axietheaxolotl** [comment]
image

**Comment by capsaicinz** [comment]
output_BU2NYc

**Comment by Kapu1178** [comment]
OH GOD OH FUCK

**Comment by capsaicinz** [comment]
i'm gonna need to fix some stuff on this later because feesh wants a new description and it STILL isn't passing checks :")

File: modular_skyrat/modules/customization/game/objects/items/plushes.dm [hunk 1]
```diff
@@ -272,6 +272,16 @@
...
 	squeak_override = list('modular_skyrat/modules/emotes/sound/emotes/synth_yes.ogg' = 1,'modular_skyrat/modules/emotes/sound/emotes/synth_no.ogg' = 1)
 
+/obj/item/toy/plush/lizzyplush
+	name = "Odd lizzy plushie"
+	desc = "A plushie made in the image of a certain, strange lizard. A faithful roboticist, and terrible father. Something about the pose unsettles you."
+	icon = 'modular_skyrat/master_files/icons/obj/plushes.dmi'
+	icon_state = "plushie_lizzy"
+	inhand_icon_state = "plushie_lizzy"
+	attack_verb_continuous = "wehs"
+	attack_verb_simple = "weh"
```
Comment (linnpap) [inline modular_skyrat/modules/customization/game/objects/items/plushes.dm]:
```suggestion
	attack_verb_continuous = list("wehs")
	attack_verb_simple = list("weh")
```
should get the integration test to past

**Review by linnpap — CHANGES_REQUESTED** [review]

**Review by GoldenAlpharex — APPROVED** [review]

**Comment by GoldenAlpharex** [comment]
The Icon Diff bot had a stroke with that animation, hopefully it's not broken.

**Pull request merged**